src/winetricks

```diff
@@ -699,16 +705,19 @@ winetricks_wintounix()
 w_pathconv()
 {
     case "$W_PLATFORM" in
-        windows_cmd)
+        windows_cygwin)
             # for some reason, cygpath turns some spaces into newlines?!
             cygpath "$@" | tr '\012' '\040' | sed 's/ $//'
             ;;
-        *)
+        windows_windows) w_die "FIXME: w_pathconv for $1 not implemented" ;;
+        nix_redox) w_die "FIXME: w_pathconv for $1 not implemented" ;;
```

Comment:
Not needed.

Reply:
elaborate? 

> windows_windows) w_die "FIXME: w_pathconv for $1 not implemented" ;;
This will trigger only if shell is executed on windows kernel and i was told that there is a 'hacky' way to get shell on native windows + sanitization doesn't hurt i believe

Reply:
It may not exist, it's not supported, and it clutters up the code.

Reply:
Seems to be possible to run shell without subsystem and cygwin: 

Referencing https://stackoverflow.com/a/21580885

Referencing https://youtu.be/0AiqegwnskE?t=107 (http://win-bash.sourceforge.net/)